requirements.txt

```diff
@@ -1,5 +1,6 @@
 gitpython
 pyyaml
+conda ==4.1.12
 conda-build-all
 conda-build ==2.0.6
-requests !=2.12.0
+requests ==2.11.1 
```

Comment:
@lbdreyer Hmmm looks like it ... 

We've pinned to using `2.11.1` internally and it seems to be the difference between the last PR that worked and this one. Seems that we're accruing debt ...